argusclient/dashboardtags.py

```diff
@@ -141,6 +141,6 @@ def AREA_CHART(*args, **kwargs):
     return _CHART(type='stackarea', *args, **kwargs)
 
 
-def STATUS_INDICATOR(name, hi, lo):
+def STATUS_INDICATOR(*args, **kwargs):
```

Comment:
Yes, the flexibility helps, but only when there are many parameters, so before we go this route, there are two options that may better suite the need here:
- If there are only handful of them, then we could specify each of them with default values.

`def STATUS_INDICATOR(name, hi, lo, p=None, q=None, r=None):
`
- If there are many options, but only a handful are mostly used, then we could leave the current ones and add the flexibility at the end.

`def STATUS_INDICATOR(name, hi, lo, *args, **kwargs):
`

What do you think?